tools/clang/unittests/HLSL/DxilContainerTest.cpp

```diff
@@ -810,10 +810,10 @@ TEST_F(DxilContainerTest, CompileWhenOKThenIncludesSignatures) {
       "; COLOR                    0   xyzw        1     NONE   float   xyzw\n"; // should read '1' in register
     if (hlsl::DXIL::CompareVersions(m_ver.m_ValMajor, m_ver.m_ValMinor, 1, 5) < 0) {
       std::string start(s.c_str(), strlen(expected_1_4));
-      VERIFY_ARE_EQUAL_STR(expected_1_4, start.c_str());
+      EXPECT_STREQ(expected_1_4, start.c_str());
```

Comment:
Isn't this what you want `VERIFY_ARE_EQUAL_STR` to resolve to on non-win32?  Are we switching all the macros now?

Reply:
I actually think we should invert this. We’ve talked about wanting to remove the dependency on TAEF for our unit tests on Windows, so we should probably replace all the TAEF-style macros with the native GoogleTest macros. Right now we have adapter macros going both ways. Someday we can hopefully get rid of all of those.

Reply:
We may wish to change all the code away from TAEF macros, but this doesn't seem like the right PR for that.

You didn't change other macros nearby and EXPECT_STREQ actually has a worse mapping for Win32 at the moment.  It would have been better to map VERIFY_ARE_EQUAL_STR to EXPECT_STREQ on linux for minimal change improvement in this scenario.

Mapping on Windows (HlslTestUtils.h):
```
#define EXPECT_STREQ(a, b) VERIFY_ARE_EQUAL(0, strcmp(a, b))
```

Reply:
I'll revert these changes but I think you're splitting hairs here.

This test was failing on Linux. It provided output that I didn't find useful in the log, so I fixed that as part of debugging. If you're saying I can't fix one usability issue without fixing all occurrences of the usability issue that more or less equates to "we can't incrementally improve things".

Reply:
That's a strawman, and not what I was saying at all.

The change will drop the string details that would have been present before when running on the Windows platform.

So it's an improvement only in this case for Linux (because on Linux we don't map the TAEF logging settings for the test), and a regression for this case on Windows - we would no longer log the string contents that didn't match, so now, we'd have the same problem on Windows that we ran into on Linux.

A slightly different change could have helped on Linux even more broadly, without regressing the functionality on Windows.

Therefore, I didn't see a good justification for this specific change, and the regression it would cause for the Windows path.